docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -91,7 +85,9 @@ The snap options for setting environment variable uses the the following format:
 where:
 
 * `<app>` is the name of the app (service, executable)
-* `<env-var>` is a lowercase, dash-separated mapping to uppercase, underscore-separate environment variable name (e.g. `x-y`->`X_Y`). The reason for such mapping is that uppercase and underscore characters are not supported as config keys for snaps.
+* `<env-var>` is a lowercase, dash-separated mapping from the uppercase, underscore-separate environment variable name (e.g. `x-y`->`X_Y`). The reason for such mapping is that uppercase and underscore characters are not supported as config keys for snaps.
+
+We call these the *config options*, because they a have `config` prefix for the variable names.
```

Comment:
Both are called config options "because they a have config prefix for the variable names". Config options is a subset of app options. Maybe we are better off removing the statement.

Reply:
Both are called config options "because they a have config prefix for the variable names". Config options is a subset of app options. Maybe we are better off removing the statement.

Reply:
Remove the definition of the term as it seem to have added confusion.